Kernel/Arch/aarch64/init.cpp

```diff
@@ -45,7 +46,12 @@ extern "C" void exception_common(TrapFrame const* const trap_frame)
         dbgln("elr_el1: {:x}", trap_frame->elr_el1);
         dbgln("tpidr_el1: {:x}", trap_frame->tpidr_el1);
         dbgln("sp_el0: {:x}", trap_frame->sp_el0);
+
+        auto esr_el1 = Kernel::Aarch64::ESR_EL1::read();
+        dbgln("esr_el1: EC(0b{:b}) IL(0b{:b}) ISS(0b{:b}) ISS2(0b{:b})", esr_el1.EC, esr_el1.IL, esr_el1.ISS, esr_el1.ISS2);
```

Comment:
We can let the formatter handle the `0b` prefix
```suggestion
        dbgln("esr_el1: EC({:#b}) IL({:#b}) ISS({:#b}) ISS2({:#b})", esr_el1.EC, esr_el1.IL, esr_el1.ISS, esr_el1.ISS2);
```

Reply:
Ah thanks, wasn't aware of that. Fixed!